release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,62 @@
 
-|Permission common|`@backstage/plugin-permission-common`|0.7.9|Identity and access management
+== Support for Azure Kubernetes Services (AKS)
 
-|Permission node|`@backstage/plugin-permission-node`|0.7.17|Identity and access management
+You can now install and use {product} on an AKS cluster. 
 
-|Permission react|`@backstage/plugin-permission-react`|0.4.16|Identity and access management
 
-|Proxy backend|`@backstage/plugin-proxy-backend`|0.4.4|Continuous deployment
-
-|Scaffolder frontend|`@backstage/plugin-scaffolder`|0.15.1|Continuous deployment
-
-|Scaffolder backend|`@backstage/plugin-scaffolder-backend`|1.18.1|Continuous deployment
-
-|Search|`@backstage/plugin-search`|1.4.1|Continuous deployment
-
-|Search backend|`@backstage/plugin-search-backend`|1.4.6|Continuous deployment
-
-|Search backend module PostgreSQL|`@backstage/Plugin-search-backend-module-pg`|0.5.15|Continuous deployment
-
-|Search backend node|`@backstage/plugin-search-backend-node`|0.2.10|Continuous deployment
-
-|Search react|`@backstage/plugin-search-react`|1.7.1|Continuous deployment
-
-|TechDocs|`@backstage/plugin-techdocs`|1.8.0|Continuous deployment
-
-|TechDocs backend|`@backstage/plugin-techdocs-backend`|1.8.0|Continuous deployment
-
-|User settings|`@backstage/plugin-user-settings`|0.7.11|Identity and access management
-
-|===
-
-== Software templates in {product}
-
-{product} provide software templates that enable you to create components. These templates come pre-equipped with features like loading code skeletons, incorporating variables into templates, and facilitating template publication to platforms like GitHub or GitLab.
-
-With the software templates, you can create a new repository on GitHub or GitLab and upload the code sample for the new application. Following that, you can register the application in the {product-short} catalog. 
-
-Currently, {product} offers templates for .Net, Python, Go, Spring Boot, Node.js, and Quarkus.
-
-== Ability to monitor the {product} instances
-
-The {product} provides a `/metrics` endpoint, providing Prometheus metrics about your {product-short} application. You can use the `/metrics` endpoint to monitor your {product-short} instance using Prometheus and Grafana.
-
-== Ability to change the theme in {product}
-
-You can select a theme for the application using *Settings* in the {product} application. Within the application, you can select from *Light*, *Dark*, or *Auto* settings for the main color and logo.
+== Support for viewing installed plugins using the web interface 
 
+You can now use the {product-short} web interface to view a table of plugins that are installed. This feature uses the `dynamic-plugins-info` frontend component, which generates a table of plugins that are currently installed in {product}. You can apply client-side sorting, filtering, and pagination on the plugins table.
 
 
+== Supported plugins in {product} {product-version}
```

Comment:
Are we not listing new plugins added in this release?

Reply:
@shipsing I see your point in relation to this section of the RNs. 

A more general question is if we need to identify to customers which **new plugins** have been added in each release, either in the RNs or in the Admin Guide table of dynamic plugins.

@kadel @davidfestal Could you advise please?